src/app/features/genes/components/gene-nominated-targets/gene-nominated-targets.component.html

```diff
@@ -76,7 +76,7 @@ <h1 class="h2">Nominated Target List</h1>
           [columns]="columns"
           [searchTerm]="searchTerm"
           [gctLink]="{ nominations: nominations.join(',') }"
-          gctLinkTooltip="Use Agora's Gene Comparison Tool to compare all nominated genes in this list."
+          gctLinkTooltip="Use Agora's Gene Comparison Tool to compare all nominated genes in this list"
```

Comment:
This should also be reworded as a call to action: "Compare all nominated genes in this list using Agora’s Gene Comparison Tool"

Reply:
Fixed missing copy